t/dataset.t

```diff
@@ -121,74 +124,93 @@ transactionally {
         is @row[4], 'Input comment', "Input has right comment";
     }
 
+    subtest 'Branching' => {
+        subtest 'load dataset' => {
+            my $name = 'BranchTest';
+            ok $dataset = Agrammon::DB::Dataset.new(:$name, :$user), "Create dataset object";
+            ok $dataset.load, "Dataset id=$dataset-id loaded";
+        }
+
+        subtest 'store and load branch data' => {
+            # TODO: cleanup data sent by the frontend
+            my $data-json = '{
+                "dataset_name":"TestRegionalBranched",
+                "instance":"Branched",
+                "vars":[
+                    "Livestock::Poultry[]::Housing::Type::housing_type",
+                    "Livestock::Poultry[]::Housing::Type::manure_removal_interval"
+                ],
+                "options":{
+                    "Livestock::Poultry[]::Housing::Type::housing_type":[
+                        "manure belt with manure belt drying system",
+                        "manure belt without manure belt drying system",
+                        "deep pit",
+                    "deep litter"],
+                    "Livestock::Poultry[]::Housing::Type::manure_removal_interval":[
+                        "less than twice a month",
+                        "twice a month",
+                        "3 to 4 times a month",
+                        "more than 4 times a month",
+                        "once a day",
+                        "no manure belt"
+                    ]
+                },
+                "data":[
+                    ["0","5","0","0","0","0"],
+                    ["0","0","10","7","0","0"],
+                    ["13","20","0","0","0","22"],
+                    ["0","15","0","8","0","0"]
+                ],
+                "tdata":[
+                    ["manure belt with manure belt drying system","0","5","0","0","0","0"],
+                    ["manure belt without manure belt drying system","0","0","10","7","0","0"],
+                    ["deep pit","13","20","0","0","0","22"],
+                    ["deep litter","0","15","0","8","0","0"]
+                ],
+                "voptions":[
+                    ["manure belt with manure belt drying system","manure belt without manure belt drying system","deep pit","deep litter"],
+                    ["less than twice a month","twice a month","3 to 4 times a month","more than 4 times a month","once a day","no manure belt"]
+                ]
+            }';
+            my $data = from-json $data-json;
+
+            ok $dataset.store-branch-data(
+                $data<vars>, $data<instance>, $data<options>, $data<data>
+            ), "Store branch data";
```

Comment:
This would pass with just a true value, which'd be better than the string.

Reply:
Changed it to `lives-ok`.